overlord/snapshotstate/snapshotstate.go

```diff
@@ -317,12 +317,6 @@ func List(ctx context.Context, st *state.State, setID uint64, snapNames []string
 	return sets, nil
 }
 
-// XXX: Something needs to cleanup incomplete imports. This is conceptually
```

Comment:
was this just done in a different pr or is this no longer a concern?

Reply:
Yes, was addressed before but we forgot to remove this comment.